GOVERNANCE.md

```diff
@@ -157,15 +182,20 @@ artifacts.  Such contributions are typically made by directly opening
 a [pull request (PR) against the main project
 repository](https://github.com/chapel-lang/chapel/pulls) containing
 and explaining their proposed changes.  The commits within such PRs
-must be signed to certify conformance with the [Developer Certificate
-of Origin
+must be signed by the developer to certify conformance with the
+[Developer Certificate of Origin
 (DCO)](https://github.com/chapel-lang/chapel/blob/main/.github/CONTRIBUTING.md).
 
 As noted above, such PRs must be reviewed and merged by a core
```

Comment:
You don't define what is meant by "core contributor".  I'm assuming it's synonymous with "committer" as defined above.

Reply:
Good catch, that's old terminology that I've become blind to.  I fixed one other related mention of "core" as well in the latest commits (the second of which was a spelling/formatting fix).